packages/client/components/Dashboard/DashSidebar.tsx

```diff
@@ -23,7 +23,6 @@ const Nav = styled('nav')<{isOpen: boolean}>(({isOpen}) => ({
   [makeMinWidthMediaQuery(Breakpoint.DASH_BREAKPOINT_WIDEST)]: {
     bottom: 0,
     left: 0,
-    position: 'fixed',
```

Comment:
Dashboard content doesn't take into account the sidebar's position when it's `fixed`